ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+#!/apps/bio/software/anaconda2/envs/hcp/bin/python3
+
+# Please note, this is an early version. All the verbose-stuff will be reduced at later stages.
+# TODO: functions for copy finished data to web folder
+# TODO: Information to slims?
+# TODO: What to do with original folders?
+
+import os
+import glob
+import smtplib
+import argparse
+import subprocess
+from email.message import EmailMessage
+
+mail_to = ""  # Change this, and uncomment argument before EOF
```

Comment:
Read from config file

Reply:
Ok, no problems to do it that way. Where should such a config-file live? Do we want it in a config-file in etc (like /etc/custom-scripts/ALU_wrapper.conf) or in the same dir as the script (usually in a dir like /usr/local/sbin or similar? Perhaps also the possibility to override the config-file with flags would be nice?